src/main/java/Sleepy.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+public class Sleepy {
+    private static final String LOGO = """
+                ____________________________________________________________
+                  ____   _      ____   ____  ____   __    __
+                 / ___| | |    | ___| | ___| |    \\ \\ \\  / /
+                 \\___ \\ | |    | ===| | ===| | D  |  \\ V  /
+                  ___) || |___ | |__  | |__  |   _/   |  |
+                 |____/ |____| |____| |____| |__|     |__|
+                """;
+    private static final String LINE_SEPARATOR = "____________________________________________________________\n";
+    public static void main(String[] args) {
+        System.out.println(LOGO);
+        String greeting = getGreeting();
+        System.out.println(greeting);
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        TaskManager taskManager = new TaskManager(Storage.loadTasks());
+        line = in.nextLine();
+        //checks for all the keywords
+        while (!line.equals("bye")) {
```

Comment:
There is quite a bit of magic strings / numbers usage in the entire codebase, perhaps reduce it?